mubench.reviewsite/src/Models/Review.php

```diff
@@ -26,6 +26,11 @@ public function tags()
         return $this->belongsToMany(Tag::class, 'review_tags', 'review_id', 'tag_id');
     }
 
+    public function misuse()
+    {
+        return $this->belongsTo(Misuse::class);
```

Comment:
Does this use the 'misuse_id' field automatically? If it doesn't we should ensure it does, to prevent managing the same link twice!